### DISCUSSION TITLE
macos+obs，外部桌面无法显示这个插件

### DISCUSSION THREAD

**Discussion by pincman** [xfgryujk/blivechat#246] [提问]
在obs画布里正常显示，但是电脑桌面不显示，最小化obs后就看不到弹幕了

<img width="1912" alt="image" src="https://github.com/user-attachments/assets/7c202c1a-7ea9-4768-9d1a-a909cb77cce2">

**Comment by xfgryujk** [comment]
唉，我都不知道怎么有人问这种问题的

**Reply by pincman** [reply]
新手直播，想请教一下原因

**Reply by xfgryujk** [reply]
要不你研究一下插件系统，把[原生UI](https://github.com/xfgryujk/blivechat/discussions/166)插件在Mac上跑起来。虽然我没试过但理论上可以

**Reply by pincman** [reply]
这个插件系统貌似只有win的，有点糊涂了，我看B站视频里很多人好像直接可以在外面显示的，我也不知啥原因就是显示不出来

**Reply by xfgryujk** [reply]
> 新手直播，想请教一下原因

OBS的预览画面和你的屏幕就没关系，你在OBS里加个字幕屏幕上会显示吗

**Reply by pincman** [reply]
也就是这个插件不能钉在桌面上放滚动弹幕，主要是OBS里看的么？

**Reply by pincman** [reply]
我感觉是不是有啥chrome插件什么的。。。看不来了哈哈

**Reply by SSSSSSSan** [reply]
> 这个插件系统貌似只有win的，有点糊涂了，我看B站视频里很多人好像直接可以在外面显示的，我也不知啥原因就是显示不出来

直接在外面回复是用了别人打包的blivechatGUI项目 或者直接用直播姬的悬浮窗
这个在obs里添加的是给观众看的

**Reply by pincman** [reply]
> > 这个插件系统貌似只有win的，有点糊涂了，我看B站视频里很多人好像直接可以在外面显示的，我也不知啥原因就是显示不出来
> 
> 直接在外面回复是用了别人打包的blivechatGUI项目 或者直接用直播姬的悬浮窗 这个在obs里添加的是给观众看的

那我明白了，感谢，主要是看了这个图，有点差异
<img width="1182" alt="image" src="https://github.com/user-attachments/assets/418eb85d-7b0c-4f02-aace-07d8da9e6fc9">
